custom_components/tuya_local/devices/jiahong_et44w_thermostat.yaml

```diff
@@ -0,0 +1,194 @@
+name: Jiahong ET-44W thermostat
+entities:
+  - entity: climate
+    dps:
+      - id: 1
+        type: boolean
+        name: hvac_mode
+        mapping:
+          - dps_val: false
+            value: "off"
+          - dps_val: true
+            constraint: mode
+            conditions:
+              - dps_val: "smart"
+                value: auto
+              - dps_val: "hold"
+                value: heat
+              - dps_val: "holiday"
```

Comment:
I don't think this is correct. Holiday mode is not cool. It is probably better to make these preset_modes and have hvac_mode just switch between heat and off.